src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,50 @@
+package controller;
+
+import model.LottoGame;
+import model.LottoResult;
+import service.LottoService;
+import util.LottoNumberGenerator;
+import view.InputView;
+import view.ResultView;
+
+import java.util.List;
+
+public class LottoController {
+    private final LottoService lottoService;
+
+    public LottoController(LottoService lottoService) {
+        this.lottoService = lottoService;
+    }
+
+    public void run() {
+        List<LottoGame> lottoGames;
+        InputView inputView = new InputView();
+        ResultView resultView = new ResultView();
+        int purchasePrice = inputView.getPurchasePrice();
+
+        int numberOfGames = lottoService.calculateNumberOfGames(purchasePrice);
+        lottoGames = lottoService.generateLottoGames(numberOfGames);
+
+        inputView.printNumberOfGames(numberOfGames);
+        inputView.printLottoGames(lottoGames);
+
+        String lottoNumberString = inputView.getLottoWinningNumbers();
+        List<Integer> lottoNumbers = lottoService.getLottoNumbers(lottoNumberString);
+        LottoResult lottoResult = new LottoResult(lottoNumbers);
+
+        for (LottoGame lottoGame : lottoGames) {
+            lottoResult.addGame(lottoGame);
+        }
+
+        List<String> lottoResultStat = lottoResult.getStatStrings();
+        resultView.printStat(lottoResultStat);
```

Comment:
```suggestion
        Lottos lottos = lottoService.create(InputView.price());
        OutputView.printLottos(lottos);

        Result result = lottoService.play(lottos, InputView.winnings());
        OutputView.print(result);
```

- 위 뼈대 코드에 맞춰서 한번 진행해보시겠어요? (클래스 이름은 달라도 무방합니다.)
  - 동시성이 요구사항은 아니라, View model은 유틸성 클래스로 분리했어요.
  - 객체에서 값을 가져와서 로직을 수행하는게 아니라, 메시지를 요청하도록 작성해보세요. getter를 계속 사용하고 있다면 의심해보세요.
  - 출력을 위해 값을 하나씩 다 받아와서 전달하기 보다는 Value Object를 도출해보세요.